.github/workflows/publish-prs.yml

```diff
@@ -0,0 +1,29 @@
+# File generated by the GradleUtils `setupGitHubActionsWorkflows` task, avoid modifying it directly
+# The template can be found at https://github.com/neoforged/GradleUtils/blob/a65628b0c89dec60b357ce3f8f6bfa62934b8357/src/actionsTemplate/resources/.github/workflows/publish-prs.yml
+
+name: Publish PRs to GitHub Packages
+
+on:
+  workflow_run:
+    workflows: [Check PRs]
+    types:
+      - completed
+  issue_comment:
+    types:
+      - edited
+  pull_request_target:
+    types:
+      - opened
+
+permissions:
+  packages: write
+
+jobs:
+  publish-prs:
+    if: true
+    uses: neoforged/actions/.github/workflows/publish-prs.yml@main
```

Comment:
Please configure the [uploader workflow name](https://github.com/neoforged/action-pr-publishing/blob/main/action.yml#L14) too.